inst/rmarkdown/templates/mdpi/skeleton/skeleton.Rmd

```diff
@@ -128,45 +160,83 @@ Numbered lists can be added as follows:
 
 The text continues here.
 
+## Figures, Tables and Schemes
+
 All figures and tables should be cited in the main text as Figure 1, Table 1, etc.
 
 \begin{figure}[H]
 \centering
-\includegraphics[width=3 cm]{logo-mdpi}
+\includegraphics[width=3 cm]{Definitions/logo-mdpi}
 \caption{This is a figure, Schemes follow the same formatting. If there are multiple panels, they should be listed as: (\textbf{a}) Description of what is contained in the first panel. (\textbf{b}) Description of what is contained in the second panel. Figures should be placed in the main text near to the first time they are cited. A caption on a single line should be centered.}
 \end{figure}
 
 
-\begin{table}[H]
-\caption{This is a table caption. Tables should be placed in the main text near to the first time they are cited.}
-\centering
-%% \tablesize{} %% You can specify the fontsize here, e.g.  \tablesize{\footnotesize}. If commented out \small will be used.
+The MDPI table float is called specialtable and can be specified in Latex as follows:
+
+\begin{specialtable}[H]
+\caption{This is a table caption. Tables should be placed in the main text near to the first time they are~cited.\label{tab1}}
+%%% \tablesize{} %% You can specify the fontsize here, e.g., \tablesize{\footnotesize}. If commented out \small will be used.
 \begin{tabular}{ccc}
 \toprule
 \textbf{Title 1}	& \textbf{Title 2}	& \textbf{Title 3}\\
 \midrule
-entry 1		& data			& data\\
-entry 2		& data			& data\\
+Entry 1		& Data			& Data\\
+Entry 2		& Data			& Data\\
 \bottomrule
 \end{tabular}
-\end{table}
+\end{specialtable}
 
-This is an example of an equation:
+Note that using default Markdown tables will put these in a normal table environment which may cause issues with numbering. To pass the 'specialtable' environment you can use knitr::kable (and the following arguments get you approximately the same style as the journal's.
```

Comment:
That is great !

you can add a link to the doc too maybe ? 
https://bookdown.org/yihui/rmarkdown-cookbook/kable.html#customize-latex-tables